js/views/userPage/Store.js

```diff
@@ -0,0 +1,404 @@
+        passesFilter = false;
+      }
+
+      if (this.filter.category !== 'all' &&
+        md.get('category').indexOf(this.filter.category) === -1) {
+        passesFilter = false;
+      }
+
+      const searchTerm = this.filter.searchTerm;
+
+      if (searchTerm &&
+        md.searchTitle.indexOf(searchTerm) === -1 &&
+        md.searchDescription.indexOf(searchTerm) === -1) {
+        passesFilter = false;
+      }
+
+      if (this.filter.shipsTo &&
+        !md.shipsTo(this.filter.shipsTo)) {
+        passesFilter = false;
+      }
+
+      return passesFilter;
+    });
+
+    return new Listings(models);
+  }
+
+  /**
+   * Based on the sortBy filter, will appropriatally set the
+   * comparator value on the given collection.
+   */
+  setSortFunction(col) {
+    if (!col) {
+      throw new Error('Please provide a collection.');
+    }
+
+    if (this.filter.sortBy) {
+      if (this.filter.sortBy === 'PRICE_ASC') {
+        col.comparator = (a, b) => {
+          if (a.get('price').amount > b.get('price').amount) {
+            return 1;
+          } else if (a.get('price').amount < b.get('price').amount) {
+            return -1;
+          }
+
+          return 0;
+        };
+      } else if (this.filter.sortBy === 'PRICE_DESC') {
+        col.comparator = (a, b) => {
+          if (a.get('price').amount < b.get('price').amount) {
+            return 1;
+          } else if (a.get('price').amount > b.get('price').amount) {
+            return -1;
+          }
+
+          return 0;
+        };
+      } else if (this.filter.sortBy === 'NAME_ASC') {
+        col.comparator = (a, b) => {
+          if (a.get('title').toLocaleLowerCase() > b.get('title').toLocaleLowerCase()) {
+            return 1;
+          } else if (a.get('title').toLocaleLowerCase() < b.get('title').toLocaleLowerCase()) {
+            return -1;
+          }
+
+          return 0;
+        };
+      } else if (this.filter.sortBy === 'NAME_DESC') {
+        col.comparator = (a, b) => {
+          if (a.get('title').toLocaleLowerCase() < b.get('title').toLocaleLowerCase()) {
+            return 1;
+          } else if (a.get('title').toLocaleLowerCase() > b.get('title').toLocaleLowerCase()) {
+            return -1;
+          }
+
+          return 0;
+        };
+      }
+    }
+  }
+
+  renderListings(col) {
+    if (!col) {
+      throw new Error('Please provide a collection.');
+    }
+
+    if (!this.storeListings) {
+      this.storeListings = new StoreListings({
+        collection: col,
+        storeOwner: this.model.id,
+      });
+    } else {
+      this.storeListings.collection = col;
+    }
+
+    this.setSortFunction(col);
+    col.sort();
+
+    if (!$.contains(this.$listingsContainer[0], this.storeListings.el)) {
+      this.$listingsContainer.empty()
+        .append(this.storeListings.el);
+    }
+
+    const listingCountContent =
+      `<span class="txB"><span>${col.length}</span> listing` +
```

Comment:
I'll be doing translations in one big sweep in the more-store branch later. On that PR, feel free to go all out with any untranslated strings. I did make a note on this string so I don't miss it.

Re: extra span - no clue why, I've removed it.